contents/haskell/Haskell.md

```diff
@@ -0,0 +1,195 @@
+# Introduction to Haskell
+
+Authors: Thenaesh Elango
+
+
+# Overview
+
+Haskell is a purely functional programming language with strong, static,
+inferred typing. It is one of the languages that feature the
+[second-order lambda calculus](https://en.wikipedia.org/wiki/System_F)
```

Comment:
Don't think a newcomer to the language will appreciate the significance of that, or be able to understand the wikipedia page.